tests/test_shows.py

```diff
@@ -129,6 +129,13 @@ def test_show_comment():
     assert got.comment('Test Comment Data').get('comment')
 
 
+def test_collection_progress():
+    show = TVShow('Game of Thrones')
+    assert show.progress is None
```

Comment:
could we add some mock data so that we can actually make assertions about the results of these methods?

Reply:
how did you make those mock data?

Need these:

```
Missing mock for GET https://api-v2launch.trakt.tv/shows/game-of-thrones/progress/collection
Missing mock for GET https://api-v2launch.trakt.tv/shows/game-of-thrones/progress/watched
```

Reply:
added cc0be4a